MODULE.bazel

```diff
@@ -70,6 +70,12 @@ use_repo(
     "tart",
 )
 
+rules_ios_bazel_version_deps = use_extension("//rules:module_extensions.bzl", "rules_ios_bazel_version_deps")
+use_repo(
+    rules_ios_bazel_version_deps,
+    "rules_ios_bazel_version",
+)
```

Comment:
Since Skylib Bazel version API can only be used in macros, could we instead determine Bazel 7 similarly to rules_apple: https://github.com/bazelbuild/rules_apple/blob/f8fa4d9a32556ece0f7eee0ebdf997ca25bdc855/apple/internal/transition_support.bzl#L46